addons/crm/wizard/crm_lead_to_opportunity_views.xml

```diff
@@ -5,6 +5,7 @@
         <field name="model">crm.lead2opportunity.partner</field>
         <field name="arch" type="xml">
             <form string="Convert to Opportunity">
+                <field name="force_assignment" invisible="1"/>
```

Comment:
is it used somewhere ? or it's added to the view to have it in memory during the wizard execution ? :thinking:

Reply:
Indeed not used ... could be removed. It is not in any triggers.